docs/one-click-integrations/bitbucket-integration.md

```diff
@@ -0,0 +1,163 @@
+
+    ![Pulse Bitbucket integration set up successfully](images/bb-ok.png)
+
+    If there was an error please [contact support](mailto:pulsesupport@codacy.com).
+
+1.  Choose the workspace that you want to connect the integration to. You can only select a workspace where you have owner permissions.
+
+    ![Choosing the Bitbucket workspace](images/bb-workspace.png)
+
+1.  Click **Complete setup**.
+
+    If there is an error please [contact support](mailto:pulsesupport@codacy.com).
+
+## Automatic deployment detection using merged pull requests {: id="deployment-detection-merged-pr"}
+
+!!! note
+    <!--NOTE Limitation from Bitbucket integration V1-V2-->Pulse uses your merged pull requests to automatically detect your deployments. Other strategies to detect deployments will be available with a future version of Pulse.
+
+-   Pulse considers a deployment every merged pull request that **targets the default branch** of the repository.
+-   The deployment date is the timestamp when the corresponding pull request is merged.
+-   The set of changes in a deployment is the list of commits in the corresponding pull request. Pulse correctly tracks your changes even if you [squash or fast-forward the commits when merging the pull request](https://support.atlassian.com/bitbucket-cloud/docs/merge-a-pull-request/#Merge-strategies), since Pulse processes all the original commits before any changes to the Git history.
+-   Pulse associates all the Bitbucket user groups of the author of a merged pull request with the corresponding deployment, excluding the user groups with less than two members. Pulse only takes the changes to Bitbucket user groups into account on pull requests merged after those changes.
+
+## Collected data
+
+The table below lists the data that the Bitbucket integration collects from your Bitbucket workspace, together with:
+
+-   The mapping between the data collected from Bitbucket and the [Pulse data model](https://ingestion.pulse.codacy.com/v1/api-docs#tocs_event)
+-   The metrics that Pulse displays on the dashboards, calculated using the collected data
+
+<table>
+<thead>
+<tr>
+<th><strong>Data collected from Bitbucket</strong></th>
+<th><strong>Mapping to Pulse data model</strong></th>
+<th><strong>Used in</strong></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+    <td>Pull request commits</td>
+    <td>
+        <p>Changes:</p>
+        <ul>
+            <li><code>change_id</code>: commit UUID</li>
+            <li><code>time_created</code>: commit author date<sup><a href="#commit-author-date">1</a></sup></li>
+            <li><code>system</code>: repository name</li>
+        </ul>
+    </td>
+    <td>Lead time for changes on the <a href="../../metrics/accelerate/">Accelerate Overview dashboard</a></td>
+</tr>
+<tr>
+    <td>Pull requests</td>
+    <td>
+        <p>Deployments:</p>
+        <ul>
+            <li><code>deploy_id</code>: pull request ID</li>
```

Comment:
This is a bit different from GH. Since there is no unique ID for a pull request in BB, we use the pull request number (unique within a repository) and the timestamp of when the PR is merged.

Not sure if it's worth going into too much detail.

Reply:
We may just update the wording to be more generic, such as `unique pull request identifier`

Reply:
```suggestion
            <li><code>deploy_id</code>: unique pull request identifier</li>
```